docs/Manual/Administration/Starter/Removal.md

```diff
@@ -0,0 +1,57 @@
+# ArangoDB Starter Removal Procedure
+
+This procedure is intended to remove a machine from a cluster
+(that was started with the ArangoDB _Starter_).
+
+It is possible to run this procedure while the machine is still running
+or when it has already been removed.
+
+It is not possible to remove machines that have an agent on it!
```

Comment:
How about adding a comment here saying that if one has to move the starter to different hardware, one can use the recovery procedure described in the other section?